pkg/collector/runner/expvars/workers.go

```diff
@@ -12,11 +12,12 @@ import (
 )
 
 const (
-	// Top-level key for this expvar
-	workersExpvarKey = "Workers"
+	// WorkersExpvarKey - Top-level key for this expvar
+	WorkersExpvarKey = "Workers"
+	// InstancesExpvarKey - Nested key for the instances expvar
+	InstancesExpvarKey = "Instances"
```

Comment:
Instead of exporting the keys, what do you think of adding Getter functions in the package ?
That way we don't leak the implementation outside of the package (that it's using expvar, with a key FOO within a map named BAR, using whatever type, etc)

Reply:
This is an interesting request. My assumption was that the contents of expvars (which are, after all, global state) represent a kind of contract in itself. Providing a getter would imply that the structure of the expvar data is an internal implementation detail, when really that structure and those data are external state.

I'm open to being convinced otherwise :)

Reply:
That's a fair point !
Although the expvars are exposed to the outside, I don't consider them part of an API / "contract", we expose whatever variable we need to, but I don't think we'd be worried about removing one for example
So in that regard I would argue it's best to keep the details like actual underlying data types and path to access one variable internal to that package